src/ol/renderer/webgl/TileLayer.js

```diff
@@ -528,10 +525,10 @@ class WebGLTileLayerRenderer extends WebGLLayerRenderer {
     }
 
     this.helper.finalizeDraw(
-      frameState, 
-      tileLayer.styleVariables_.glBlendEquation,
-      tileLayer.styleVariables_.glBlendFuncSRC,
-      tileLayer.styleVariables_.glBlendFuncDST
+      frameState,
```

Comment:
`Number(undefined)` would now be `NaN`. 
You can typecast the values instead. And please use a lowercase letter for the start of the method name.
```js
/** @type {GLenum|undefined} */ (tileLayer.getStyleVariables().glBlendEquation),
```